src/app/components/configure-wallet/configure-wallet.component.ts

```diff
@@ -113,20 +107,34 @@ export class ConfigureWalletComponent implements OnInit {
     this.walletService.informNewWallet();
   }
 
+  async connectLedgerByBluetooth() {
+    this.ledgerService.enableBluetoothMode(true);
+    await this.importLedgerWallet();
+  }
+
+  async connectLedgerByUsb() {
+    this.ledgerService.enableBluetoothMode(false);
+    await this.importLedgerWallet();
+  }
+
   async importLedgerWallet(refreshOnly = false) {
     // Determine status of ledger device using ledger service
     this.notifications.sendInfo(`Checking for Ledger device...`, { identifier: 'ledger-status', length: 0 });
     await this.ledgerService.loadLedger(true);
     this.notifications.removeNotification('ledger-status');
 
     if (this.ledger.status === LedgerStatus.NOT_CONNECTED) {
-      return this.notifications.sendWarning(`No ledger device detected, make sure it is connected and you are using Chrome/Opera`);
+      return this.notifications.sendWarning(`No ledger device detected.  Make sure the Nano app is running on the Ledger.  Restart the Nano App if the error persists`);
```

Comment:
think that's the last occurrence of it

```suggestion
      return this.notifications.sendWarning(`No Ledger device detected.  Make sure the Nano app is running on the Ledger.  Restart the Nano App if the error persists`);
```